Tiltfile

```diff
@@ -0,0 +1,11 @@
+local_resource(
```

Comment:
The docs don't mention tilt, can you add that if we're going to commit a Tiltfile?